### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by zunairkhan811**
- Specie column added

speciescolumnadded

- Insert data to animals tabel

insertnewdata

- Add unspecified to column specie

unspecifiedandrollback;

- Different queries and their screenshot

setspecieendwithmon

setemptyspecietopokemon

createsavepointandweight-1

rollbacktosp1andupdateweightandcommit

deleteandrollbackwholedata

deletegreaterthan1stjan

rollbacktosp1andupdateweightandcommit

queryquestions1

queryquestion2

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,11 @@ INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8);
 INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+INSERT INTO animals VALUES (5,'Charmander','2020-02-08',0,false,11);
+INSERT INTO animals VALUES (6,'Plantmon','2021-11-15',2,true,5.7) ;
+INSERT INTO animals VALUES (7,'Squirtle','1993-04-02',3,false,12.13) ;
+INSERT INTO ANIMALS VALUES (8,'Angemon','2005-06-12',1,true,45) ;
+INSERT INTO ANIMALS VALUES (9,'Boarmon','2005-06-07',7,true,20.4) ;
+INSERT INTO ANIMALS VALUES (10,'Blossom','1998-10-13',3,true,17) ;
+INSERT INTO ANIMALS VALUES (11,'Ditto','2022-05-14',4,true,22) ;
```
Comment (abdulhamiid) [inline data.sql]: According to the project requirements, we were instructed to add the following data:
Kindly take note of the negative signs in some of the values 👀
image

- It was later in the transaction that all the values were update to positive 💪🛠

image

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @zunairkhan811 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use database transactions.
- ✔️Modify and delete data in SQL.
- ✔️Prepare complex queries that answer analytical questions.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (zunairkhan811) [reply data.sql]: Thanks for the suggestions. Your suggestions have been implemented. 😃

**Review by Munsa1 — APPROVED** [review]
Hi @zunairkhan811, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 
descriptive Pr :heavy_check_mark: 
Good Communication :heavy_check_mark: 

## Optional suggestions
### Important :red_circle: 
- [x] It would have been nice if you could update your screenshots to reflect the changes made

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zunairkhan811** [comment]
Hi! @Munsa1, Thanks for your suggestion. The screenshot has been updated. 😉

**Pull request merged**